composer.json

```diff
@@ -1,37 +1,28 @@
 {
     "name": "dhii/espresso-interface",
-    "description": "The interfaces for Espresso.",
+    "description": "The interfaces for Espresso: a PHP expression builder and evaluation library.",
     "type": "library",
     "license": "MIT",
     "authors": [
         {
-            "name": "Dhii",
+            "name": "Dhii Team",
             "email": "development@dhii.co"
         }
     ],
     "minimum-stability": "dev",
     "require": {
         "php": "^5.3.9 | ^7.0",
```

Comment:
@mecha, one way to fix this could be to file a new PR for the release branch, and merge again after the fix.